lib/discourse_chat_integration/provider/slack/slack_provider.rb

```diff
@@ -211,7 +211,7 @@ def self.set_slack_thread_ts(topic, channel, value)
         created_at: Time.zone.now,
         updated_at: Time.zone.now,
       },
-      unique_by: %i[topic_id name],
+      unique_by: :index_topic_custom_fields_on_topic_id_and_slack_thread_id,
```

Comment:
this isn't the same index, is it?

Reply:
Yes it is :)
https://github.com/discourse/discourse-chat-integration/blob/main/db/migrate/20211202120030_add_unique_index_to_slack_thread_ts.rb